src/pluto.py

```diff
@@ -518,10 +523,13 @@ def generate_graph(v1, v2):
         st.write(df)
         st.info(
             """
-         In addition to looking at the number of lots with a changed value, it’s important to look at the magnitude of the change.
+         The aggregate graph provides insights into the magnitude of the change, complementing the mismatch graph's functionality of showing the number of lots with a changed value.
          For example, the mismatch graph for finance may show that over 90% of lots get an updated assessment when the tentative roll is released.
-         The aggregate graph may show that the aggregated sum increased by 5%. Totals for assessland, assesstot, and exempttot should only change in February and June.
-         Pay attention to any large changes to residential units (unitsres).
+         The aggregate graph may show that the aggregated sum increased by 5% compared with the previous version.\n
+         Special Notices:\n
+         1. Y-axis represents percent change over the previous version. \n
+         2. Totals for assessland, assesstot, and exempttot should only change in February and June.\n
+         3. Pay attention to any large changes to residential units (unitsres).
```

Comment:
This writeup is good, I made a few small changes. I think we should keep in the part about how some values should only change in February and June?